### ISSUE TITLE
Online positions disappearing at 00:00z

### ISSUE THREAD

**Issue by Dunkstormen** [Vatsim-Scandinavia/home#39]
Whilst converting the BookingComponent to TSX from JSX I encountered that online positions would disappear at 00:00z. Most likely due to the dateMap where todays date is the first in the list, which causes it to not be added as the date doesn't match.

A solution could be to start the dateMap from the previous day so positions online before 00:00z still show up as eg. Tuesday... despite it being Wednesday etc.

**Comment by blt950** [comment]
If I understand you correctly, if someone logs on prior to 00.00z, their position disappears from the live atc?

**Comment by Dunkstormen** [comment]
> If I understand you correctly, if someone logs on prior to 00.00z, their position disappears from the live atc?

You're absolutely right! I can try resolve using the above solution. Though I'm currently working on another bug in the attempt to convert the component to TSX where online positions don't show on initial load.

**Comment by blt950** [comment]
If you are able to look into resolving this when able, that'd be greatly appreciated 😄
